tesseract_collision/test/collision_sphere_sphere_unit.cpp

```diff
@@ -80,12 +80,8 @@ void runTest(tesseract::DiscreteContactManagerBase& checker)
   //////////////////////////////////////
   // Test when object is in collision
   //////////////////////////////////////
-  tesseract::ContactRequest req;
-  req.link_names.push_back("sphere_link");
-  req.link_names.push_back("sphere1_link");
-  req.contact_distance = 0.1;
-  req.type = tesseract::ContactRequestType::CLOSEST;
-  checker.setContactRequest(req);
+  checker.setActiveCollisionObjects({"sphere_link", "sphere1_link"});
```

Comment:
So is `req.link_names` now named `setActiveCollisionObjects()`? All former uses now directly translate to the latter? I'm trying to keep the [hybrid examples](https://github.com/Jmeyer1292/hybrid_planning_experiments) from ROSCON updated and these changes just broke everything.